source/Octopus.Cli/Commands/Deployment/DeploymentCommandBase.cs

```diff
@@ -305,7 +294,7 @@ private async Task<IReadOnlyList<DeploymentResource>> DeployToEnvironments(Proje
             {
                 var tenantPromotions = releaseTemplate.TenantPromotions.Where(
                     tp => tp.PromoteTo.Any(
-                        promo => promo.Name.Equals(environmentName, StringComparison.CurrentCultureIgnoreCase))).Select(tp => tp.Id).ToArray();
+                        promo => promo.Name.Equals(environmentName, StringComparison.OrdinalIgnoreCase))).Select(tp => tp.Id).ToArray();
```

Comment:
Why did this change?
Will this cause issues on say, french servers?

Reply:
I just noticed these few were inconsistent with the majority of resource name comparisons. We usually use Ordinal, which I agree with: it's generally preferable for computer identifiers, which these mostly are. We want a quick and exact match, unaffected by locale settings on the hosts. I am thinking we probably don't want a Project called "Grüße" to be identifiable as "grüsse" on one machine/user but not for another. CurrentCulture is more appropriate for human-language-centric string processing, like sorting for display.